File: Programming Basics with Python/While-цикъл/account_balance.py

insert = input()

total = 0.0
while insert != "NoMoreMoney":
    money = float(insert)
    if money < 0:
        print("Invalid operation!")
        break
    
    total += money
    print(f"Increase: {money:.2f}")
    insert = input()

print(f"Total: {total:.2f}") 

File: Programming Basics with Java/While-loop/AccountBalance.java
import java.util.Scanner;

public class AccountBalance 
{
    public static void main(String[] args) 
    {
         Scanner input = new Scanner(System.in);

         double balance = 0.0;

         while (true)
         {
            String command = input.nextLine();

            if (command.equals("NoMoreMoney"))
            {
                break;
            }

            double money = Double.parseDouble(command);
            if (money <= 0)
            {
                System.out.println("Invalid operation!");
                break;
            }
            else
            {
                balance += money;
                System.out.printf("Increase: %.2f\n", money);
            }
         }

        System.out.printf("Total: %.2f", balance);
    }
}
